include/range/v3/utility/addressof.hpp

```diff
@@ -0,0 +1,69 @@
+// Range v3 library
+//
+//  Copyright Eric Niebler 2013-present
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+//
+#ifndef RANGES_V3_UTLITY_ADDRESSOF_HPP
+#define RANGES_V3_UTLITY_ADDRESSOF_HPP
+
+#include <memory>
+#include <type_traits>
+#include <concepts/concepts.hpp>
+#include <range/v3/detail/config.hpp>
+
+namespace ranges
+{
+    /// \cond
+    namespace detail
+    {
+#ifdef __cpp_lib_addressof_constexpr
+        using std::addressof;
+#else
+        namespace test {
+            struct ignore {
+                template <typename T> ignore(T&&) { }
+            };
+
+            ignore operator&(ignore);
```

Comment:
I suspect if you defined this function, the travis errors will go away.

```suggestion
            inline ignore operator&(ignore) { return {}; }
```

Also, use `detail::ignore_t` which range-v3 already provides, and give the `test` namespace a more descriptive name.